### ISSUE TITLE
Account/password recovery with e-mail doesn't work 

### ISSUE THREAD

**Issue by fminierf** [jhipster/jhipster-online#364]
**Describe the bug**
E-mail recovery for JHipster online account doesn't work. When I provide my e-mail address an e-mail with account/password recovery instructions never comes.

**To Reproduce**
1. Go to signin form of jhister website
2. Click on 'Did you forget your password?'
3. fill in Email field with account email and click on 'Reset'
4. Message 'Check your emails for details on how to reset your password.' is displayed
5. No instructions received on email account

**Expected behavior**
A mail should be received on email account with instructions to reset password

**Additional context**
The problem occurs regularly #281 #308 #323 #350 ...

**Comment by Ederagp** [comment]
this still doesn't work !!!

**Comment by jdubois** [comment]
Hey sorry I just noticed this, don't hesitate to ping me directly for something related to production! I'm the only one with access to the credit card, which is needed for the email provider :-)
Let me have a look

**Comment by jdubois** [comment]
Here's the stack trace on the server:

```
org.thymeleaf.exceptions.TemplateInputException: Error resolving template [mail/passwordResetEmail], template might not exist or might not be accessible by any of the configured Template Resolvers
at org.thymeleaf.engine.TemplateManager.resolveTemplate ( [org/thymeleaf.engine/TemplateManager.java:869](https://console.cloud.google.com/debug?referrer=fromlog&file=org%2Fthymeleaf.engine%2FTemplateManager.java&line=869&project=jhipster-online) )
at org.thymeleaf.engine.TemplateManager.parseAndProcess ( [org/thymeleaf.engine/TemplateManager.java:607](https://console.cloud.google.com/debug?referrer=fromlog&file=org%2Fthymeleaf.engine%2FTemplateManager.java&line=607&project=jhipster-online) )
at org.thymeleaf.TemplateEngine.process ( [org/thymeleaf/TemplateEngine.java:1098](https://console.cloud.google.com/debug?referrer=fromlog&file=org%2Fthymeleaf%2FTemplateEngine.java&line=1098&project=jhipster-online) )
at org.thymeleaf.TemplateEngine.process ( [org/thymeleaf/TemplateEngine.java:1059](https://console.cloud.google.com/debug?referrer=fromlog&file=org%2Fthymeleaf%2FTemplateEngine.java&line=1059&project=jhipster-online) )
at org.thymeleaf.TemplateEngine.process ( [org/thymeleaf/TemplateEngine.java:1048](https://console.cloud.google.com/debug?referrer=fromlog&file=org%2Fthymeleaf%2FTemplateEngine.java&line=1048&project=jhipster-online) )
at io.github.jhipster.online.service.MailService.sendEmailFromTemplate ( [io/github.jhipster.online.service/MailService.java:109](https://console.cloud.google.com/debug?referrer=fromlog&file=io%2Fgithub.jhipster.online.service%2FMailService.java&line=109&project=jhipster-online) )
at io.github.jhipster.online.service.MailService.sendPasswordResetMail ( [io/github.jhipster.online.service/MailService.java:129](https://console.cloud.google.com/debug?referrer=fromlog&file=io%2Fgithub.jhipster.online.service%2FMailService.java&line=129&project=jhipster-online) )
at io.github.jhipster.async.ExceptionHandlingAsyncTaskExecutor.lambda$createWrappedRunnable$1 ( [io/github.jhipster.async/ExceptionHandlingAsyncTaskExecutor.java:78](https://console.cloud.google.com/debug?referrer=fromlog&file=io%2Fgithub.jhipster.async%2FExceptionHandlingAsyncTaskExecutor.java&line=78&project=jhipster-online) )

**Comment by jdubois** [comment]
Fixed in #383
